### PR TITLE
Devex refactor interactor calling interactor to test

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCases/users/getJudgeInSectionInteractor.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ * getJudgeInSectionInteractor - returns the judge user for a given section
+ *
+ * @param {object} applicationContext the application context
+ * @param {object} obj the options object
+ * @param {string} obj.section the section to fetch the judge from
+ * @returns {User} the judge user in this section provided
+ */
+
+exports.getJudgeInSectionInteractor = async (
```
Comment (halprin) [inline shared/src/business/useCases/users/getJudgeInSectionInteractor.js:16]: Add a test for this usecase?

File: shared/src/business/useCases/caseNote/getUserCaseNoteInteractor.test.js [hunk 2]
```diff
@@ -42,12 +44,12 @@ describe('Get case note', () => {
...
   it('throws an error if the entity returned from persistence is invalid', async () => {
-    applicationContext.getCurrentUser.mockReturnValue(mockJudge);
+    applicationContext.getCurrentUser.mockReturnValue(new User(mockJudge));
     applicationContext
       .getPersistenceGateway()
       .getUserCaseNote.mockResolvedValue(omit(MOCK_NOTE, 'userId'));
-    applicationContext.getUseCases.mockReturnValue({
-      getJudgeForUserChambersInteractor: () => mockJudge,
+    applicationContext.getUseCaseHelpers.mockReturnValue({
+      getJudgeInSectionHelper: () => mockJudge,
```
Comment (halprin) [inline shared/src/business/useCases/caseNote/getUserCaseNoteInteractor.test.js:52]: Thanks!

**Review by halprin — COMMENTED** [review]
I think we're just missing one test here.

File: shared/src/business/useCases/caseNote/getUserCaseNoteInteractor.test.js (hunk 2 above)
Comment (codyseibert) [reply shared/src/business/useCases/caseNote/getUserCaseNoteInteractor.test.js:52]: nice catch ty

**Review by halprin — APPROVED** [review]

**Pull request merged**